cylc/rose/entry_points.py

```diff
@@ -40,7 +41,15 @@ def pre_configure(srcdir: Path, opts: 'Values') -> dict:
         # nothing to do here
         return {}
 
-    # load the Rose config
+    # If we are validating against source, load saved CLI options
+    # from previous install, as saved in the rose-suite-cylc-install.conf
+    if (
+        getattr(opts, 'against_source', False)
```

Comment:
It doesn't look like the `--against-source` option was ever accessed from cylc-rose, so we ~*don't* have any compatibility issues to worry about which is good news.